src/app/Shared/Services/CommandChannel.service.tsx

```diff
@@ -35,17 +35,18 @@ export class CommandChannel {
 
     fromFetch(`${this.apiSvc.authority}/health`)
       .pipe(
-        concatMap(resp => from(resp.json())))
-      .pipe(
-        concatMap(jsonResp => {
-            if((jsonResp.dashboardAvailable == true) && (jsonResp.datasourceAvailable == true)) 
-              {return forkJoin([getDatasourceURL, getDashboardURL])}
-            else {this.logError('Grafana configuration', 'URLs unavailable');
-                  return jsonResp}}))
+        concatMap(resp => from(resp.json())), 
+        concatMap((jsonResp: any) => {
+          if ((jsonResp.dashboardAvailable == true) && (jsonResp.datasourceAvailable == true)) {
+            return forkJoin([getDatasourceURL, getDashboardURL])
+          } else {
+              return throwError(new Error('ULRs unavailable'));
```

Comment:
s/ULR/URL/

Could this error include details about which URL(s) in particular are not available?

Reply:
I added specific messages depending on which is missing.